.circleci/config.yml

```diff
@@ -2,33 +2,36 @@ version: 2
 jobs:
   build:
     machine:
-      image: ubuntu-1604:202004-01
+      image: ubuntu-1604:202004-01 
     working_directory: ~/bigtable-autoscaler
     steps:
       - checkout
+      - run: 
+          name: Install Maven
+          command:
+            sudo apt-get update && sudo apt-get install maven
```

Comment:
I would have thought you need a `-y` on the install. guess not?

Reply:
what is -y?

Reply:
ah found it

Reply:
I can add to future proof lol?

Reply:
if you want to future-proof, replace `apt-get` with `apt`. Honestly, I don't really care